### ISSUE TITLE
OpenDet training from scratch

### ISSUE THREAD

**Issue by milosacimovic** [Topdu/OpenOCR#118]
Hi,

Could you please provide more information on how the shared OpenDet model RepSVTR-DB was trained?

What dataset and training strategy were used for RepSVTR-DB that is shared through the repo?
How would one train it from scratch to reproduce the results?
Are the backbone weights resumed from ImageNet pretraining?
Is there a pretraining stage involved before finetuning on a specific dataset?  I saw that many works first pretrain on SynthText for couple hundred epochs before finetuning on a benchmark set e.g. ICDAR15 etc.

Thank you for sharing this wonderful project with the community.

**Comment by Harry-KIT** [comment]
Hi @Topdu 
I am also interested in the dataset from which openocr_det_repvit_ch.pth came from.

**Comment by Topdu** [comment]
The dataset consists of existing publicly available data, including synthetic data in both Chinese and English, publicly available real data, and some pseudo-labels generated by existing OCR tools.
We did not load any pre-trained models and trained the model from scratch.

**Comment by Harry-KIT** [comment]
Hi @Topdu, thanks for the information about your dataset. Could you please share:

Which specific synthetic datasets were used (Chinese and English)?
Names of the public datasets you incorporated
Which OCR tools generated the pseudo-labels

If possible, please provide links to these resources.
Thank you.
